stereotype/model.py

```diff
@@ -144,19 +144,21 @@ def resolve_extra_types(cls) -> Set[Type[Model]]:
         return set()
 
     def __getitem__(self, key):
-        if key not in self.__slots__:
-            raise KeyError(key)  # Avoids getting methods via getitem
-        return getattr(self, key)
+        try:
+            return getattr(self, key)
```

Comment:
This would also get methods, I don't think that's good. I'll see if I can find a better way.